models/instances/library.json

```diff
@@ -1,5 +1,5 @@
 {
-  "serializationFormatVersion": "1",
```

Comment:
I wouldn't do this yet -- there might be more changes.
I already know of one (meta)-change: This field (`serializationFormatVersion`) is currently spec'd as "integer" but will change to "string"

Reply:
In the TS and Java implementations this is already a string. I guess we're slightly ahead of the updating of the specification?

Also: these files are not leading in any way.